src/install/lockfile/bun.lock.zig

```diff
@@ -751,14 +751,15 @@ pub const Stringifier = struct {
             );
         }
 
-        // TODO(dylan-conway)
-        // if (meta.libc != .all) {
-        //     try writer.writeAll(
-        //         \\"libc": [
-        //     );
-        //     try Negatable(Npm.Libc).toJson(meta.libc, writer);
-        //     try writer.writeAll("], ");
-        // }
+        if (meta.libc != .all and meta.libc != .none) {
```

Comment:
wouldn't we want to serialize either if it's `.all` or if it's `.none` because they're different?